gci/migrations/0001_initial.py

```diff
@@ -0,0 +1,101 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.11.7 on 2017-12-09 04:23
+from __future__ import unicode_literals
+
+from django.db import migrations, models
+import django.db.models.deletion
+
+
+class Migration(migrations.Migration):
+
+    initial = True
+
+    dependencies = [
+    ]
+
+    operations = [
+        migrations.CreateModel(
+            name='Category',
+            fields=[
+                ('identifier', models.IntegerField(primary_key=True, serialize=False)),
+                ('name', models.CharField(max_length=100)),
+            ],
+        ),
+        migrations.CreateModel(
+            name='Mentor',
+            fields=[
+                ('id', models.AutoField(auto_created=True, primary_key=True, serialize=False, verbose_name='ID')),
+                ('email_id', models.EmailField(max_length=254)),
+            ],
+        ),
+        migrations.CreateModel(
+            name='Organization',
+            fields=[
+                ('identifier', models.IntegerField(default=0, primary_key=True, serialize=False)),
+                ('name', models.CharField(max_length=100)),
+                ('logo', models.ImageField(blank=True, upload_to='')),
+                ('url', models.URLField(blank=True)),
+                ('summary', models.TextField(default='', max_length=300)),
+            ],
+        ),
+        migrations.CreateModel(
+            name='ProgramYear',
+            fields=[
+                ('id', models.AutoField(auto_created=True, primary_key=True, serialize=False, verbose_name='ID')),
+                ('start_date', models.DateField()),
+                ('last_date', models.DateField()),
+            ],
+        ),
+        migrations.CreateModel(
+            name='Student',
+            fields=[
+                ('identifier', models.IntegerField(default=0, primary_key=True, serialize=False)),
+                ('display_name', models.CharField(max_length=100)),
+                ('program_year', models.ManyToManyField(to='gci.ProgramYear')),
+            ],
+        ),
+        migrations.CreateModel(
+            name='Tag',
+            fields=[
+                ('id', models.AutoField(auto_created=True, primary_key=True, serialize=False, verbose_name='ID')),
+                ('name', models.CharField(max_length=100)),
+            ],
+        ),
+        migrations.CreateModel(
+            name='Task',
+            fields=[
+                ('id', models.AutoField(auto_created=True, primary_key=True, serialize=False, verbose_name='ID')),
```

Comment:
auto_created?

Reply:
I am not doing it manually, `id` this field is automatically getting create when I am running migrations?

Reply:
this is because you havent specified an identifier in the data model; a unique ID exist in the API reference.
It needs to be the primary key.

Reply:
Should I add another field as identifier?